### PR TITLE
Freebase Location API 

### PR DESCRIPTION
**Pull request by varunagrawal**
Added 3 videos to the README on how to build a small knowledge graph from Google IO 2014.

### CODE REVIEW

**Comment by varunagrawal** [comment]
Have added a function called lookup() to location.py to lookup locations in the Freebase graph.
Can be called by importing freebase.location

File: freebase/location.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    """
+    Look up location entity from Freebase given 'query' keywords.
+    Return list of locations that have the highest score given search query.
+    """
+    service_url = 'https://www.googleapis.com/freebase/v1/search'
+    
+    ## These are the params that go into the get request
+    params = {
+        'query': query,
+        'filter':'(any type:/location/location)'    # filter lets us specify which property to filter our search results on. See Freebase Metasearch.
```
Comment (pviolette3) [inline freebase/location.py:15]: Nice. Filter looks useful.

File: freebase/location.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    
+    url = service_url + '?' + urllib.urlencode(params)
+    response = json.loads(requests.get(url).text)
```
Comment (pviolette3) [inline freebase/location.py]: opt (change if you want): Can pass `requests.get` parameters directly, `requests.get(service_url, params=params)` http://docs.python-requests.org/en/latest/user/quickstart/#passing-parameters-in-urls

File: README.md [hunk 3]
```diff
@@ -46,3 +46,10 @@ a relationship between Atlanta and Georgia that has type "contains" (representin
 
 If processing raw wikipedia pages proves too difficult, we can use the [Wikidata entity API](https://www.wikidata.org/w/api.php?action=help&modules=wbgetentities) to resolve our named entities.
 For example, see [this gist](https://gist.github.com/edsu/4681747)
+
+Resoures:
+---------
+Build A Small Knowledge Graph - Google IO 2014
+* [Part 1](https://www.youtube.com/watch?v=W9pRpSW_KqA)
```
Comment (pviolette3) [inline README.md:53]: Sweet! Thanks :+1:

**Comment by pviolette3** [comment]
Cool! This looks good, I had one minor comment with using requests, which you can do if you want, or you can just merge now. Looks good!

**Comment by varunagrawal** [comment]
Updated. :smile:

**Comment by pviolette3** [comment]
Cool. Go ahead and merge :+1:

**Pull request closed without merging**